src/fileioc/fileioc.h

```diff
@@ -382,6 +382,14 @@ uint8_t ti_RclVar(const uint8_t var_type, const char *var_name, void **data_stru
  */
 bool ti_ArchiveHasRoom(uint24_t num_bytes);
 
+
+/**
+ * Set routine to run after a garbage collect.
+ * @param routine Routine to run following a garbage collect. If this is 0, the default handler will be used.
```

Comment:
Once the default handler is a no-op, it's probably clearer to just say that 0 will clear any handler set.

Reply:
Might as well say `NULL` instead of `0`, too.